x-pack/metricbeat/module/activemq/broker/_meta/docs.asciidoc

```diff
@@ -0,0 +1,12 @@
+Broker Metricset retrieves the JMX key `org.apache.activemq:brokerName=*,type=Broker` using Jolokia. It exposes the following metrics:
```

Comment:
Could you extend this description a little bit? More than the underlying retrieval method, users will be also interested in know what is it about. ie Explain this will retrieve metrics from the brokers

Reply:
I rephrased this paragraph. Feel free to review it!